packages/app-desktop/integration-tests/models/Sidebar.tsx

```diff
@@ -0,0 +1,47 @@
+
```

Comment:
This doesn't need to be a `.tsx` file — it should be renamed to have a `.ts` extension.